Meta/CMake/vcpkg/overlay-ports/skia/fix-freebsd.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/BUILD.gn b/BUILD.gn
+index 72d9a23..9c0532e 100644
+--- a/BUILD.gn
++++ b/BUILD.gn
+@@ -26,7 +26,8 @@ import("gn/ios.gni")
+ 
+ # Skia public API, generally provided by :skia.
+ config("skia_public") {
+-  include_dirs = [ "." ]
++  include_dirs = [ ".", "/usr/local/include" ]
++  lib_dirs = [ "/usr/local/lib" ]
+ 
+   defines = [
+     "SK_CODEC_DECODES_BMP",
+diff --git a/gn/toolchain/BUILD.gn b/gn/toolchain/BUILD.gn
+index 85858b5..8d6a0ae 100644
+--- a/gn/toolchain/BUILD.gn
++++ b/gn/toolchain/BUILD.gn
+@@ -315,7 +315,7 @@ template("gcc_like_toolchain") {
+         rspfile = "{{output}}.rsp"
+         rspfile_content = "{{inputs}}"
+         rm_py = rebase_path("../rm.py")
+-        command = "$shell python3 \"$rm_py\" \"{{output}}\" && $ar rcs {{output}} @$rspfile"
++        command = "$shell python3 \"$rm_py\" \"{{output}}\" && $ar rcs {{output}} `cat $rspfile`"
```

Comment:
this patch looks strange. does the default shell mangle the response file syntax? does the version of gn in FreeBSD's ports not properly create a response file given this template? or is this an llvm-ar/ar version not supporting the response file syntax issue?

Reply:
since it works on your machine we can go with this, but I suspect that using 15-CURRENT with llvm-ar will be the approach needed by an updated ports tree build.